src/GraphingImpl/Mocks/GraphingOptions.h

```diff
@@ -251,6 +252,19 @@ namespace MockGraphingImpl
             m_boxColor = Graphing::Color();
         }
 
+        virtual Graphing::Color GetGridColor() const
+        {
+            return m_gridColor;
+        }
+        virtual void SetGridColor(const Graphing::Color& value)
```

Comment:
nit: spaces between these methods

Reply:
The current style in this file has like methods grouped without spacing and the groups are spaced. I followed the spacing pattern that is already present.